portal-api/src/v1/controllers/mandatoryCriteria.controller.js

```diff
@@ -26,10 +26,15 @@ const findOneMandatoryCriteria = async (version, callback) => {
 };
 
 exports.create = async (req, res) => {
-  const collection = await db.getCollection('mandatoryCriteria');
-  const mandatoryCriteria = await collection.insertOne(req.body);
+  // MC insertion on non-production environments
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
It might be safer to instead list the environments we're happy for this to run on.

If production was ever renamed, or if another production-like environment was added these endpoints would be available on it by default. It could be easy to miss locking it down in future.

Reply:
NODE_ENV is always declared, other envs list is large so decided to target prod environment only.